index.js

```diff
@@ -29,6 +29,7 @@ program
     "-t, --temperature <temperature>",
     "Temperature for the chat completions"
   )
+  .option("--token-usage", "Show prompt and completion token usage")
```

Comment:
We can use the shorthand `-tu` flag for this since `-t` is already used for temperature!

Reply:
That's a great idea! Updated